easybuild/easyconfigs/p/packmol/packmol-v20.2.2-intel-2020a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'packmol'
+version = 'v20.2.2'
+
+homepage = 'http://m3g.iqm.unicamp.br/packmol'
+description = "Packing Optimization for Molecular Dynamics Simulations"
+
+toolchain = {'name': 'intel', 'version': '2020a'}
```

Comment:
This easyconfig file should be removed, since we have another with just `iccifort` as toolchain?